docs/plugins/ingest-geoip.asciidoc

```diff
@@ -203,3 +203,18 @@ Which returns:
 }
 --------------------------------------------------
 // TESTRESPONSE
+
+[[ingest-geoip-settings]]
+===== Settings
+
+The geoip processor supports the following settings:
+
+`ingest.geoip.cache_enabled`::
+
+    Whether to enable caching of results. Defaults to `false`.
```

Comment:
After discussion with @martijnvg I disabled it by default but I'm happy to change that.